extension/data/deinflect.dat

```diff
@@ -333,4 +336,34 @@ imperative negative
 り	る	640	24
 れ	る	640	23
 れ	れる	384	24
-ろ	る	384	23
\ No newline at end of file
+ろ	る	384	23
+です	だ	8320	13
+だった	だ	8320	14
+でした	だ	8320	6
+ではない	だ	8320	15
+じゃない	だ	8320	15
+ではないです	だ	8320	1
+じゃないです	だ	8320	1
+ではありません	だ	8320	1
+じゃありません	だ	8320	1
+ではなかった	だ	8320	28
```

Comment:
Continuing from above, I think you can remove these two lines if you change the type mask for the root above.